packages/oui-datagrid/README.md

```diff
@@ -165,6 +165,13 @@ Clicked row action 1: <span ng-if="$ctrl.action1Row">{{$ctrl.action1Row.lastName
 | `property`        | string          | N/A     | yes              |                           |                        | property path used to get value from value  |
 | `sortable`        | string          | N/A     | yes              | `asc`, `desc`             | `asc` on `sortable=""` | makes a column sortable and gives the order |
 
+### rows-loader promise response
+
+| Attribute         | Type            | Binding | One-time binding | Values                    | Default             | Description                                                                                  |
+| ----              | ----            | ----    | ----             | ----                      | ----                | ----                                                                                         |
+| `data`            | array<object>   | <?      | yes              |                           |                     | rows to show                                                                                 |
+| `meta`            | object          | N/A     | yes              |                           |                     | an object containing pagination information {currentOffset, pageCount, totalCount, pageSize} |
```

Comment:
In fact, I think only `totalCount` is needed, could you check that in a datagrid?